packages/veritone-widgets/test/redux/modules/veritoneApp/saga.test.js

```diff
@@ -0,0 +1,32 @@
+import { put, call, select } from 'redux-saga/effects';
+import { cloneableGenerator } from 'redux-saga/utils';
+import { removeFunctions } from '../../../helpers/utils';
+import {
+  handleAppAuth,
+  watchAppAuth,
+} from '../../../../src/redux/modules/veritoneApp/saga';
+import * as appModule from '../../../../src/redux/modules/veritoneApp';
+import _ from 'lodash';
+import { modules } from '../../../../../veritone-redux-common/dist/bundle-es.js';
```

Comment:
veritone-redux-common is a dependency of veritone-widgets - `import { modules } from 'veritone-redux-common'`. You can` npm link` to verify it's working.